src/plugins/graph/hooks/use-plot.ts

```diff
@@ -194,11 +197,14 @@ export const usePlotResponders = (props: IPlotResponderProps) => {
   // respond to color changes
   useEffect(() => {
     return reaction(() => {
-      const colors: Record<string, string> = {};
-      const layers = Array.from(graphModel.layers);
-      const descriptions = layers.map(l => l.config.yAttributeDescriptions);
-      descriptions.forEach(desc => desc.forEach(d => colors[d.attributeID] = graphModel.getColorForId(d.attributeID)));
-      return JSON.stringify(colors);
+      if (isAlive(graphModel)) {
```

Comment:
Rather than this `isAlive()` test, better would be to use `mstReaction` which automatically disposes the reaction if the model is disposed.

Reply:
Makes sense.  I made that change.